charts/single-node-values.yml

```diff
@@ -19,3 +19,7 @@ mongodb-replicaset:
   replicaSet: rs0
   replicas: 3
 
+redis-ha:
```

Comment:
If we can get away with simply using the normal redis cache for the single node deployments, it would help to keep requirements down.

Reply:
We will need to use globals to solve that issue properly.

Reply:
Yeah this could be setup later on if we end up having resourcing issues.

Reply:
I think for single nodes having 1 sentinel is the right approach (this is how it's implemented in Federation as well). This helps if the deployment scales to multiple nodes.